docs/widgetbook-cloud/quick-start.mdx

```diff
@@ -25,10 +25,11 @@ This quick start guide will help you set up and use Widgetbook Cloud efficiently
 2. Install and configure the Widgetbook CLI.
 3. Ensure that your application's resources are accessible by configuring CORS settings for your backend services.
 
-## Step 5: Configure GitHub Webhook (Optional)
+## Step 5: Configure GitHub App (Optional)
 
-1. Retrieve your Webhook Secret from the project overview.
-2. Set up a webhook in your GitHub repository settings.
-3. Configure the webhook settings with the appropriate Payload URL, Content type, Secret, and trigger events.
+1. Initiate via your Company Profile
+2. Select your Company
+3. Start GitHub App Installation Flow
+4. Link to a Project, Connect your Repository and Sync Reviews with Pull Requests
 
 Now, you're all set to use Widgetbook Cloud to manage your widget libraries, collaborate on reviews, and streamline your development workflow.
```

Comment:
Doesn't belong here but I just noticed in the section `Signup` in the 1. step, the sentence reads: Sign up for Early Access to Widgetbook Cloud by visiting [widgetbook.io early access form] and selecting an appropriate plan. 

Here, please delete the last part of the sentence _ and selecting an appropriate plan_

Reply:
I have deleted both, but I thought having an optional sectoon for github app would help developer to understand that there are other options. Just deleted it anyway